test/net/sourceforge/kolmafia/utilities/FileUtilitiesTest.java

```diff
@@ -42,5 +42,6 @@ public void exerciseSomeMethodsForCoverage() {
     FileUtilities.copyFile(source, destination);
     assertTrue(destination.exists());
     destination.delete();
+    source.delete();
```

Comment:
I was a little concerned by this, but it doesn't seem to be changing my source tree, so I'm OK with it.

Reply:
In a test environment, everything is under test\root and AFAIK this test only deletes what it created.  That said this test looks confusing to me now and I'd like more confidence that nothing is coming from "outside".  Maybe I'll shelve this for now.